recipes/fwdpy11/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+
+export CFLAGS="-I${PREFIX}/include"
+export CXXFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+#We must force GCC on OS X
+export CC=gcc
```

Comment:
Working on it.  It seems that LLVM is getting older and older on OS X.  This code requires clang 3.8 or so on Ubuntu, which is well ahead of where OS X is, AFAIK.